test/parallel/test-constants.js

```diff
@@ -14,7 +14,7 @@ assert.ok(binding.crypto);
 
 ['os', 'fs', 'crypto'].forEach((l) => {
   Object.keys(binding[l]).forEach((k) => {
-    if (typeof binding[l][k] === 'object') { // errno and signals
+    if (k === 'errno' || k === 'signals') {
```

Comment:
Because this test fails without an explicit check for `errno` and `signals`. `process.binding('constants')` exports constants from `node_constants.cc:DefineConstants()`, while `require('constants')` exports constants from the seemingly deprecated lib/constants.js.

I haven't touched lib/constants.js because it said it was deprecated, and so now the exports don't match. That make sense?